### PR TITLE
Update to 1.18.2

### PR DESCRIPTION
**Pull request by dragonmaus**
Updated the mod for 1.18.2 (it was crashing as soon as a golem tried to come into existence).

### CODE REVIEW

File: src/main/java/com/sollace/coppergolem/OxidizableCopperButtonBlock.java [hunk 1]
```diff
@@ -13,6 +13,7 @@ protected OxidizableCopperButtonBlock(Oxidizable.OxidationLevel oxidizationLevel
         super(oxidizationLevel, settings);
     }
 
+    @SuppressWarnings("deprecation")
```
Comment (dragonmaus) [inline src/main/java/com/sollace/coppergolem/OxidizableCopperButtonBlock.java]: The changes involved in fixing this are rather invasive (as far as I can tell), so I left it out for now.

**Comment by Sollace** [comment]
Can you please separate the cleanup from simply updating to 1.18 and make a separate PR for that. I see a lot of changes here that I'm not sure are neeccessary.

**Comment by dragonmaus** [comment]
Sure thing. I had a feeling I was putting too much into one PR anyway, but wanted to get it out for your consideration.

**Comment by dragonmaus** [comment]
That should be the bare minimum for updating to 1.18.2.

**Review by Sollace — APPROVED** [review]

**Comment by Sollace** [comment]
Looks good. I will have to test it locally before I do a release, but I see nothing keeping it from being merged.

Thanks!

**Pull request merged**